#global definition 
std_msgs/String filename_reference
std_msgs/String filename_target
geometry_msgs/Point point_offset
std_msgs/Float32 theta_rad
std_msgs/UInt32 focuspos
std_msgs/UInt32 id_image
std_msgs/UInt32 id_search
std_msgs/UInt32 id_chip
---
#result definition
geometry_msgs/Point point_result
---
#feedback
std_msgs/Byte currentState
geometry_msgs/Point currentPosition
